transforms/code/license_check/python/src/license_check_transform.py

```diff
@@ -0,0 +1,144 @@
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+################################################################################
+
+
+import json
+from argparse import ArgumentParser, Namespace
+
+import pyarrow as pa
+from data_processing.data_access import DataAccess, DataAccessFactory
+from data_processing.transform import AbstractTableTransform, TransformConfiguration
+from data_processing.utils import TransformUtils, get_logger
+from transformer import AllowLicenseStatusTransformer, DenyLicenseStatusTransformer
+
+
+logger = get_logger(__name__)
+
+LICENSE_CHECK_PARAMS = "license_check_params"
+
+shortname = "lc"
+CLI_PREFIX = f"{shortname}_"
+LICENSE_COLUMN_NAME_KEY = f"{CLI_PREFIX}license_column_name"
```

Comment:
I'd like to separately the cli keys from the transform configuration keys.  THe cli prefix is something to help use when running things in pipelines to distinguish transform-specific cli parameters.  However, when it comes to. using the transform init() there is no need for this prefix.  So, the keys to init() should be, for example `license_column_name` and the cli parameter should be `lc_license_column_name`.

Reply:
Thanks for the detailed explanation.